pkg/services/libraryelements/libraryelements_patch_test.go

```diff
@@ -344,7 +345,7 @@ func TestPatchLibraryElement(t *testing.T) {
 	scenarioWithPanel(t, "When an admin tries to patch a library panel in another org, it should fail",
 		func(t *testing.T, sc scenarioContext) {
 			cmd := model.PatchLibraryElementCommand{
-				FolderID: sc.folder.ID,
+				FolderID: sc.folder.ID, // nolint:staticcheck
```

Comment:
actually, is this a sequential FolderID? it's the `ID` field on the `Folder` struct